src/fuse_ops.c

```diff
@@ -256,6 +256,10 @@ static void *unionfs_init(struct fuse_conn_info *conn, struct fuse_config *cfg)
 			exit(1);
 		}
 	}
+	
+	if (uopt.direct_io)
```

Comment:
plase make this single line ((( if (uopt.direct_io) cfg->direct_io = 1; ))) or keep the {}. also please add spaces around =.